packages/fast-usdc/test/exos/settler.test.ts

```diff
@@ -221,7 +224,13 @@ test('happy path: disburse to LPs; StatusManager removes tx', async t => {
     [],
     'SETTLED entry removed from StatusManger',
   );
-  // TODO, confirm vstorage write for TxStatus.SETTLED
+  // Flush event loop since status manager doesn't await vstorage writes
+  await new Promise(res => setTimeout(res, 1));
```

Comment:
`await eventLoopIteration();` from `@agoric/internal` is a helper you can also use

Reply:
Done